Foundation/FileManager+POSIX.swift

```diff
@@ -932,8 +932,14 @@ extension FileManager {
             if supportsStatx {
```

Comment:
Can we set `supportsStatx = false` after the `EPERM` to avoid later `_statx` calls that will fail.

Reply:
Possibly.  But it's not clear (to me) whether statx could return `EPERM` for other reasons besides the syscall being denied.  From the docs, it seems like it would return `EACCES` if the user supplies an inaccessible path, so maybe it would be okay?

Reply:
Ok lets leave it for now, it can always be added as an enhancement later if necessary.